app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -132,9 +132,12 @@ def export
       end
 
       api :GET, '/register', N_('Render Global Registration template')
+      param :organization_id, :number, required: true, desc: N_("ID of Organization")
```

Comment:
`ID of the Organization to register the host in.` and similarly below.

Also lets open an issue for textual identifiers, it would be nice to specify org name. Similarly to also Location and Hostgroup